build.rs

```diff
@@ -41,6 +41,19 @@ mod musl_reference_tests {
         "rem_pio2.rs",
         "rem_pio2_large.rs",
         "rem_pio2f.rs",
+        "remquo.rs",
+        "remquof.rs",
+        "lgamma.rs",  // lgamma passed, lgamma_r has more than 1 result
+        "lgammaf.rs", // lgammaf passed, lgammaf_r has more than 1 result
+        "frexp.rs",   // more than 1 result
+        "frexpf.rs",  // more than 1 result
+        "sincos.rs",  // more than 1 result
+        "sincosf.rs", // more than 1 result
+        "modf.rs",    // more than 1 result
+        "modff.rs",   // more than 1 result
+        "asinef.rs",  // not exists
```

Comment:
What does "not exists" mean here?

Reply:
Panics with
```
--- stderr
error: linking with `cc` failed: exit code: 1
  |
  = note: "cc" "-Wl,--as-needed" "-Wl,-z,noexecstack" "-Wl,--eh-frame-hdr" "-m64" "-nostdlib" "/home/burrbull/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-musl/lib/crt1.o" "/home/burrbull/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-musl/lib/crti.o" "-L" "/home/burrbull/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-musl/lib" "gen.gen.7rcbfp3g-cgu.0.rcgu.o" "gen.gen.7rcbfp3g-cgu.1.rcgu.o" "gen.gen.7rcbfp3g-cgu.10.rcgu.o" "gen.gen.7rcbfp3g-cgu.11.rcgu.o" "gen.gen.7rcbfp3g-cgu.12.rcgu.o" "gen.gen.7rcbfp3g-cgu.13.rcgu.o" "gen.gen.7rcbfp3g-cgu.14.rcgu.o" "gen.gen.7rcbfp3g-cgu.15.rcgu.o" "gen.gen.7rcbfp3g-cgu.2.rcgu.o" "gen.gen.7rcbfp3g-cgu.3.rcgu.o" "gen.gen.7rcbfp3g-cgu.4.rcgu.o" "gen.gen.7rcbfp3g-cgu.5.rcgu.o" "gen.gen.7rcbfp3g-cgu.6.rcgu.o" "gen.gen.7rcbfp3g-cgu.7.rcgu.o" "gen.gen.7rcbfp3g-cgu.8.rcgu.o" "gen.gen.7rcbfp3g-cgu.9.rcgu.o" "-o" "gen" "gen.ml8rp173tno6czu.rcgu.o" "-Wl,--gc-sections" "-no-pie" "-Wl,-zrelro" "-Wl,-znow" "-nodefaultlibs" "-L" "/home/burrbull/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-musl/lib" "-Wl,--start-group" "-Wl,-Bstatic" "/home/burrbull/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-musl/lib/libstd-e9655bcee52420f6.rlib" "/home/burrbull/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-musl/lib/libpanic_unwind-94a79f5396bbf06e.rlib" "/home/burrbull/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-musl/lib/libbacktrace_sys-7a0a2cd8401e3fbb.rlib" "/home/burrbull/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-musl/lib/librustc_demangle-544fe11ac5351948.rlib" "/home/burrbull/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-musl/lib/libhashbrown-2de4e4f6ac9ab0a9.rlib" "/home/burrbull/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-musl/lib/librustc_std_workspace_alloc-9b193946d0ee39d9.rlib" "/home/burrbull/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-musl/lib/libunwind-c94045c5315cd4de.rlib" "/home/burrbull/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-musl/lib/liblibc-1e84ab9d7a609c9b.rlib" "/home/burrbull/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-musl/lib/liballoc-d698a486b5d0cf97.rlib" "/home/burrbull/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-musl/lib/librustc_std_workspace_core-bf2642637f4f98fd.rlib" "/home/burrbull/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-musl/lib/libcore-82c12243ccd26b23.rlib" "-Wl,--end-group" "/home/burrbull/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-musl/lib/libcompiler_builtins-0bd15cf37b00baa7.rlib" "-static" "-Wl,-Bdynamic" "/home/burrbull/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-musl/lib/crtn.o"
  = note: gen.gen.7rcbfp3g-cgu.0.rcgu.o: In function `gen::main':
          gen.7rcbfp3g-cgu.0:(.text._ZN3gen4main17heffb9d4db471d1caE+0x58aa): undefined reference to `asinef'
          collect2: error: ld returned 1 exit status

```

Reply:
If the function doesn't exist in musl why are we exporting it from this library?

Reply:
I'll remove this one.